### ISSUE TITLE
AnyChart Stock: Tooltip and zooming keeps increasing DOM node and Memory.

### ISSUE THREAD

**Issue by dharaksavalia** [AnyChart/AnyChart#211]
I was testing https://www.anychart.com/products/anystock/gallery/Stock_Chart_Types/Multi-Series_Spline_Chart.php, and notice that when keep interacting with chart, i.e. hover over chart, tooltip is display or keep on scrolling the chart results in increase in DOM node and heap memory.
I could track the element which keeps on increasing in  DOM, have attached image for it. 
`<text>` element which child of `<g>`  which is child of `<svg>` keeps on  increasing.

**Comment by sturdystablestalwart** [comment]
Thank you for promptly notifying our team; we are testing this and will make sure to fix it as soon as this issue is confirmed.

**Comment by sturdystablestalwart** [comment]
@dharaksavalia, Yes, the issue was confirmed, and the fix is scheduled for the next release.
Thank you for your report, and if you have any additional concerns or questions, I'm here to help.

**Comment by dharaksavalia** [comment]
In meantime, do we have temporary fix or way to mitigate this issue ?
